src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -112,8 +105,7 @@ public function send(ApiClient $apiClient): PaymentResponse
 			DateTimeImmutable::createFromFormat('YmdHis', $data['dttm']),
 			new ResultCode($data['resultCode']),
 			$data['resultMessage'],
-			isset($data['paymentStatus']) ? new PaymentStatus($data['paymentStatus']) : null,
-			$data['authCode'] ?? null
+			isset($data['paymentStatus']) ? new PaymentStatus($data['paymentStatus']) : null
```

Comment:
Z dokumentace to tak vypadá, ale konkrétně se píše: `Návratové hodnoty jsou shodné s definicí popsanou u operace payment/init.` a u payment/init je paymentStatus označený jako nepovinný. Zajímalo by mě, kdy může být paymentStatus vynechaný. Nikde jsem se s tím nesetkal. Možná chyba v dokumentaci, zkusím založit issue